recipes/subby/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This license file is already included in the source distribution so it can be removed from here.